ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -328,6 +342,8 @@ private ExecutionPlan getStatementExecutionPlan(Statement statement, String stat
     if (ddlCommandTask != null) {
       try {
         return new ExecutionPlan(ddlCommandTask.execute(statement, statementText, properties));
+      } catch (KsqlException ksqlException) {
```

Comment:
Can we change this method so that it throw `KsqlException` rather than `Exception` - generally we shouldn't have methods with the signature ` blah() throws Exception` unless we have no control over what is being thrown. Even then, we should probably catch and wrap such cases so we have the context of what has gone wrong etc

Reply:
Agreed, updated it. When we did this we were still finalizing the KsqlException class. Updated it. Will have to look into other places to use KsqlException as much as possible.